src/main/java/vg/civcraft/mc/civmodcore/command/CommandHandler.java

```diff
@@ -76,5 +99,21 @@ protected void helpPlayer(Command command, CommandSender sender) {
 		sender.sendMessage(new StringBuilder().append(ChatColor.RED + "Usage: ")
 				.append(command.getUsage()).toString());
 	}
+	
+	public void msg(String msg) {
```

Comment:
it was only a helper method to send a message to the command sender but on second look, it's not really worth having. I removed it.